bindings/java/src/org/sleuthkit/datamodel/CommunicationsManager.java

```diff
@@ -1316,18 +1316,48 @@ int getAccountTypeId(Account.Type accountType) {
 	 * @return The normalized account id.
 	 */
 	private String normalizeAccountID(Account.Type accountType, String accountUniqueID) throws TskCoreException {
-		String normailzeAccountID = accountUniqueID;
+		String normalizedAccountID = accountUniqueID;
 
 		if (accountType.equals(Account.Type.PHONE)) {
-			normailzeAccountID = CommunicationsUtils.normalizePhoneNum(accountUniqueID);
+			normalizedAccountID = normalizePhoneNumber(accountUniqueID);
 		} else if (accountType.equals(Account.Type.EMAIL)) {
-			normailzeAccountID = CommunicationsUtils.normalizeEmailAddress(accountUniqueID);
+			normalizedAccountID = normalizeEmailAddress(accountUniqueID);
 		}
 
-		return normailzeAccountID;
+		return normalizedAccountID;
 	}
-
 	
+	/**
+	 * Normalize the given phone number to be suitable for storage.
+	 *
+	 * @param phoneNum The phone number to normalize
+	 *
+	 * @return The normalized phone number.
+	 */
+	private String normalizePhoneNumber(String phoneNumber) {
+		String strippedOfNonDigits = phoneNumber.replaceAll("\\D", "");
+
+		if (strippedOfNonDigits.isEmpty()) {
```

Comment:
This whole if statement doesn't really make sense to me, this is just how it was previously.
Seems like we should instead throw a TskCoreException.